modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js

```diff
@@ -142,15 +142,50 @@ export default function propsTransformer({portletNamespace, ...otherProps}) {
 			onSelect: (selectedItems) => {
 				let redirectURL = itemData?.redirectURL;
 
+				const selectedItemsJSONArray = [];
+
 				selectedItems.forEach((item) => {
-					redirectURL = addParams(
-						`${portletNamespace}contentDashboardItemSubtypePayload=${JSON.stringify(
-							item
-						)}`,
-						redirectURL
-					);
+					let entryClassNameItem;
+
+					if (item.entryClassName && item.className) {
```

Comment:
I think this conditional `if (item.entryClassName && item.className) ... else if (item.entryClassName) {` 
is not really needed, isn't it? Also because both conditionals look for the same thing.

so just look in the array if exists or not. 

```
entryClassNameItem = selectedItemsJSONArray.find(
	(object) =>
		object.entryClassName === item.entryClassName
);
```

Reply:
Yes, Totally agree with you, I don' know what I was thinking :(.

thanks for the review